### DISCUSSION TITLE
App pulled?

### DISCUSSION THREAD

**Discussion by mjreposits** [shumbo/Svadilfari#57] [General]
Is Scadilfari pulled permanently from the App Store as it seems?  Or will not receive future support.

Sad if so, I love it and it has so much potential!

**Comment by shumbo** [comment]
@mmj3445 Thank you for letting me know the app was not available on the App Store. This was not intentional and due to an issue with my Apple developer account. I resolved the issue and the app should be available again soon. I'm sorry that this happened and thanks again @mmj3445 for reporting this issue.

**Reply by mjreposits** [reply]
Np!
